File: src/node/orders/buy-order.js
const Easypost = require('@easypost/api');
const dotenv = require('dotenv');

dotenv.config({ path: '/Users/jhammond/git/easypost/easypost-tools/.env' });
const api = new Easypost(process.env.EASYPOST_PROD_API_KEY);

api.Order.retrieve('order_123...').then(order => {
    order.buy(
        'AustraliaPost', // carrier
        'ExpressPostSignature' // service
    ).then(console.log).catch(console.error)
}).catch(console.log);

File: src/python/orders/buy_order.py
import os

import easypost
from dotenv import load_dotenv

load_dotenv()
easypost.api_key = os.getenv('EASYPOST_PROD_API_KEY')

order = easypost.Order.retrieve("order_16e9b3a4c9104b408bc7103f0c7be2c7")
order.buy(carrier='FedEx', service='FEDEX_2_DAY')

print(order)
